pubspec.yaml

```diff
@@ -1,7 +1,6 @@
 name: optimizely_plugin
 description: Flutter plugin for Optimizely SDK
-version: 0.0.1
-author: Policygenius Engineering
```

Comment:
Why the author removal?

Reply:
More asking to make sure we don't have something to replace the field with to make sure we keep that #branding

Reply:
Also we should delete the commented out stuff we don't need in this file.

Reply:
Yep it's been deprecated and replaced with the verified publisher that we need to setup: https://dart.dev/tools/pub/pubspec#authorauthors